mod/host.py

```diff
@@ -1152,7 +1152,10 @@ def report_current_state(self, websocket):
             for symbol, value in pluginData['outputs'].items():
                 if value is None:
                     continue
-                websocket.write_message("output_set %s %s %f" % (pluginData['instance'], symbol, value))
+                if isinstance (value, float):
+                    websocket.write_message("output_set %s %s %f" % (pluginData['instance'], symbol, value))
+                if isinstance (value, str):
```

Comment:
It's the plugin's DSP job to replay any messages (also in other LV2 hosts).

The common practice is for the GUI to send a message to the plugin when it is opened/loaded to query the state if it's needed.